main/src/main/java/com/sedmelluq/discord/lavaplayer/source/youtube/YoutubeHttpContextFilter.java

```diff
@@ -40,8 +40,8 @@ public void onRequest(HttpClientContext context, HttpUriRequest request, boolean
 
     request.setHeader("user-agent", "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) " +
         "Chrome/76.0.3809.100 Safari/537.36");
-    request.setHeader("x-youtube-client-name", "1");
-    request.setHeader("x-youtube-client-version", "2.20191008.04.01");
+    request.setHeader("x-youtube-client-name", "ANDROID");
+    request.setHeader("x-youtube-client-version", "16.24");
```

Comment:
Changes either of these lines completely breaks the responses for me, Youtube sends me `)]}` instead of the song information. Changes both these lines back fixes it. I've yet to try if age restricted videos still play after reverting these headers. But currently its not usable when changing the headers to this.